keyboards/dz60/keymaps/spotpuff/keymap.c

```diff
@@ -0,0 +1,79 @@
+#include QMK_KEYBOARD_H
+
+//Layers
+#define _QWERTY 0
+#define _FN     1
+#define _MEDIA  2
+
+//Filler for KC_TRNS/KC_NO just so the key maps are a bit more readable
+#define ______ KC_TRNS
+#define XXXXXX KC_NO
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

Reply:
If I remove those lines I get a compile error:
keyboards/dz60/keymaps/spotpuff/keymap.c:52:3: error: ‘XXXXXX’ undeclared here (not in a function)

Same w/ _____

Reply:
Use 7 X's and _'s.